tensorflow/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/core/Dense.kt

```diff
@@ -31,6 +31,13 @@ import org.tensorflow.op.Ops
  * created by the layer, and `bias` is a bias vector created by the layer
  * (only applicable if `use_bias` is `True`).
  *
+ * __Input shape:__  N-D tensor with shape `(batch_size, ..., input_dim)`.
```

Comment:
Currently, only 2d input is supported.

Reply:
Resolved in the following commit:
https://github.com/Kotlin/kotlindl/pull/460/commits/34c394f3a6df776eddc8c1bb443aed8cb0ec5356